### DISCUSSION TITLE
Who can be the member of .NET platform organization.

### DISCUSSION THREAD

**Discussion by ShreyasJejurkar** [dotnet/runtime#50700] [Q&A]
Hi all, 
I am not sure if this is the correct place to ask this question or not, but I found the runtime repo is more active in the discussion tab than any other dotnet repo. 

The question is already in the title. I mean who can apply to become a member of the [.NET platform](https://github.com/dotnet)? 
1. Is it locked to only Microsoft employees those who work for `dotnet`? 
2. Can open source contributors who contribute to `dotnet` repo consistently can apply to become a member? 

Is there any pre-defined policy or guidelines for the same? 

Thanks!

**Comment by alexrp** [comment]
I'm not sure if there's an official policy or process, but I think anyone who's an actual maintainer (i.e. with write access) of a repo under the `dotnet` organization will be a member under the [People](https://github.com/orgs/dotnet/people) tab. Many repos under the organization are part of the [.NET Foundation](https://dotnetfoundation.org) but aren't necessarily maintained by Microsoft.

**Reply by ShreyasJejurkar** [reply]
Hmm, thanks for the reply! I have seen some people, who are part of .NET platform but they are not actual Microsoft employee, so am just wondering is there any process or something like that just like we have for .NET foundation! 

Waiting for some more response! Might be @terrajobst may have some idea!

**Reply by ShreyasJejurkar** [reply]
@terrajobst you there? Or no one knows?

**Comment by FilipToth** [comment]
Anyone can sign up to be a member of the dotnet foundation. Here’s some more information: https://dotnetfoundation.org/member/become-a-member

**Reply by ShreyasJejurkar** [reply]
FYI dotnet foundation is different organization from .NET platform!

Dotnet foundation - https://github.com/dotnet-foundation

.NET Platform - https://github.com/dotnet

I am asking for .NET platform and not dotnet organization!
